hermes/templates/base.html

```diff
@@ -0,0 +1,45 @@
+{% load static bootstrap4 %}
+<!doctype html>
+<html lang="en">
+  <head>
```

Comment:
I think all of these templates are unreachable. These were for the temporary "front end" before we had the vue JS stuff.

Reply:
So should we remove them?